python/transformations/RSS Filtering/library.json

```diff
@@ -0,0 +1,49 @@
+{
+    "libraryItemId": "c4f4c164-1e7f-4a2c-81fa-f35f55c2d9b2",
+    "name": "RSS Data Filtering",
+    "language": "Python",
+    "tags": {
+        "Complexity": ["Medium"],
+		"SDK Usage": ["Read/Write SDK"],
+        "Pipeline Stage": ["Transformation"],
+		"Technology": ["RSS"]
+    },
+    "shortDescription": "Example of how to filter and enhance RSS data",
+	"DefaultFile": "quix_function.py",
+	"EntryPoint": "build/dockerfile",
+    "RunEntryPoint": "main.py",
+    "Variables": [
+        {
+            "Name" : "input",
+            "Type" : "EnvironmentVariable",
+            "InputType" : "InputTopic",
+            "Description" : "This is the input topic for raw data",
+			"Required" : true
+        },
+		{
+            "Name" : "output",
+            "Type" : "EnvironmentVariable",
+            "InputType" : "OutputTopic",
+            "Description" : "This is the output topic for filtered and updated data",
+            "DefaultValue" : "filtered-rss",
+			"Required" : true
```

Comment:
not a blocking reason, but could you update your editor to insert spaces over tabs